M2/Macaulay2/packages/VirtualResolutions.m2

```diff
@@ -556,6 +561,6 @@ uninstallPackage "VirtualResolutions"
 restart
 installPackage "VirtualResolutions"
 restart
-needsPackage("VirtualResolutions", FileName => "./VirtualResolutions.m2")
+needsPackage("VirtualResolutions", FileName => "VirtualResolutions.m2")
```

Comment:
Why is the FileName option still needed?

Reply:
I'm not sure who needed that, but it's in the footer, so does it matter?